onnxruntime/test/providers/nnapi/nnapi_basic_test.cc

```diff
@@ -449,6 +449,16 @@ TEST(NnapiExecutionProviderTest, TestQDQConcat_UnsupportedInputScalesAndZp) {
 }
 #endif
 
+TEST(NnapiExecutionProviderTest, TestQDQGemm) {
+  RunQDQModelTest(BuildQDQGemmTestCase<uint8_t, uint8_t, uint8_t>(
+                      {2, 2} /* input_shape1 */,
+                      {2, 2} /* input_shape2 */,
+                      true /* has_bias */, 
+                      1 /* transB */),
+                  "nnapi_qdq_test_graph_gemm",
```

Comment:
Can we also test without bias, with transA == 1, and with transB == 0?

Reply:
for this, since NNAPI only supports 1. A*B'+C 2. A*B+C case, I am not adding without bias with transA=1 test case here for now.

Do we just want to test it is unsupported?  (added test case with bias and transB=0.)

Reply:
We should test with bias not being provided. The existing code in the op builder creates a tensor with zeros for the bias if not provided, so we want to hit that to make sure it works as expected.

Reply:
added a test with param - no bias+transB= 1